doc/developer/webservices.rst

```diff
@@ -12,6 +12,9 @@ Parameters
 ----------
 
 * ``version``: ``1`` or ``2 ``, API version, default is ``1``.
+* ``sets``: with kind of data we want to get, can be ``all``, ``themes``, ``group``
```

Comment:
"with kind of data" => "which kind of data" (or simply "kind of data" or "type of data")